src/renderer.ts

```diff
@@ -125,7 +129,9 @@ const renderSlide = async (
 ) => {
   slide.hidden = node.hidden;
   if (node.backgroundColor) slide.background = { fill: node.backgroundColor };
-  if (node.backgroundImage) slide.background = { path: node.backgroundImage };
+  if (node.backgroundImage) {
+      slide.background = { [node.backgroundImage.kind]: node.backgroundImage[node.backgroundImage.kind] };
+  }
```

Comment:
Ah, completely missed the object. Let me adjust a bit, our logic would end up looking something like: 

```suggestion
  if (node.backgroundImage) {
      const background = node.backgroundImage.entries[0]
      slide.background = { [background.key]: background.value };
  }
```

Reply:
Ah, right. I think it's a bit better as the more explicit current version. Explicitly managing data tends to work better with typescript and is more readable. Also I think this has the risk of assigning `kind` to background

Reply:
Got it. With this iteration, I would only have the 

{ data: string } | { path: string} type for images. 

Anything else you'd like to see in this PR, @wyozi ?

Reply:
Left one more comment. Also if you merge from master it should run tests etc